pkg/ccl/changefeedccl/helpers_test.go

```diff
@@ -224,34 +224,6 @@ func avroToJSON(t testing.TB, reg *cdctest.SchemaRegistry, avroBytes []byte) []b
 	return json
 }
 
-func assertPayloadsAvro(
```

Comment:
removed this since we're now only using Avro tests with Kafka testfeed which automatically converts the payloads to JSON. If other sinks adopt Avro in the future we can add this back or perform the conversion within the corresponding testfeed (similar to Kafka).